.github/dependabot.yml

```diff
@@ -0,0 +1,7 @@
+# https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

Comment:
Is there a good reason to run Dependabot on a timed/weekly schedule, rather than just "whenever the package is updated"?

(I suppose the answer is "because the dependencies might ~be updated~ get compromised by hackers in the meantime" :thinking:)

Reply:
That's what I have seen “everywhere else”, I haven't tried hard to find an alternative to [`schedule.interval`](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#scheduleinterval).

I think you're right about it. The whole idea of dependabot is to create pull requests to warn you of updates in dependencies. Dependencies might be updated to address security issues, in which case you have to follow up and update your repository to use only fixed versions of such dependencies. Therefore, you need to let dependabot scan dependencies on a regular basis and warn you of updates. Your only choice is “how often” as far as I can see.